runtime/dolphin/src/weights/frame_system.rs

```diff
@@ -56,79 +57,79 @@ pub trait WeightInfo {
 pub struct SubstrateWeight<T>(PhantomData<T>);
 impl<T: frame_system::Config> frame_system::WeightInfo for SubstrateWeight<T> {
     fn remark(_b: u32, ) -> Weight {
-        (1_672_000 as Weight)
+        (6_824_000 as Weight)
```

Comment:
remark became 5x more expensive?

Reply:
in v3.2.0 remark went f

```
-       fn remark(b: u32, ) -> Weight {
-               (8_427_000 as Weight)
-                       // Standard Error: 0
-                       .saturating_add((1_000 as Weight).saturating_mul(b as Weight))
+       fn remark(_b: u32, ) -> Weight {
+               (1_672_000 as Weight)
```
now it's back up. This large change went under the radar again

Reply:
From latest result:
```rust
fn remark(_b: u32, ) -> Weight {
        (5_378_000 as Weight)
    }
```